### ISSUE TITLE
HVAC Mode will be also translated

### ISSUE THREAD

**Issue by TheNexusCore** [nielsfaber/scheduler-card#202]
HA with german locale, the entity will look like this: 

```
entries:
  - <IDS>
actions:
  - service: set_preset_mode
    entity: climate.gast_heizung
    preset_mode: Schlafen
  - service: set_preset_mode
    entity: climate.gast_heizung
    preset_mode: Abwesend
next_trigger: '2020-11-05T18:00:00+01:00'
friendly_name: 'Schedule #a02900'
icon: 'mdi:calendar-clock'
```
The error resulting in HA log:
```
Logger: homeassistant.core
Source: components/knx/climate.py:159
First occurred: 15:40:23 (2 occurrences)
Last logged: 16:17:23

    Error executing service: <ServiceCall climate.set_preset_mode (c:af40b5c34fbfc145a2d4cbd18ff3015c): preset_mode=Abwesend, entity_id=['climate.gast_heizung']>
    Error executing service: <ServiceCall climate.set_preset_mode (c:12d9da3b717688e6f39d60178f9aca48): preset_mode=Abwesend, entity_id=['climate.gast_heizung']>

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/core.py", line 1464, in catch_exceptions
    await coro_or_task
  File "/usr/src/homeassistant/homeassistant/core.py", line 1483, in _execute_service
    await handler.job.target(service_call)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 204, in handle_service
    await self.hass.helpers.service.entity_service_call(
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 482, in entity_service_call
    future.result()  # pop exception if have
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 664, in async_request_call
    await coro
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 519, in _handle_entity_call
    await result
  File "/usr/src/homeassistant/homeassistant/components/knx/climate.py", line 159, in async_set_preset_mode
    knx_operation_mode = HVACOperationMode(PRESET_MODES_INV.get(preset_mode))
  File "/usr/local/lib/python3.8/enum.py", line 315, in __call__
    return cls.__new__(cls, value)
  File "/usr/local/lib/python3.8/enum.py", line 617, in __new__
    raise ve_exc
ValueError: None is not a valid HVACOperationMode
```

**Comment by nielsfaber** [comment]
LOL!

**Comment by TheNexusCore** [comment]
I dont think a "lol" will solve this problem :)

**Comment by nielsfaber** [comment]
no you can expect an update for this soon.
i'm just surprised something like this slipped through

**Comment by nielsfaber** [comment]
I just created a new release [v1.9.4](https://github.com/nielsfaber/scheduler-card/releases/latest) that fixes this bug.
Thanks a lot for reporting and sorry for the inconvenience.

Make sure to delete or edit your existing schedules, such that the wrong parameters will be removed from your entities.

**Comment by TheNexusCore** [comment]
I installed version 1.9.4 and it works perfekt! :)

One thing to mention, in my eyes this is no issue: In the scheduler itself, the word is now also in english. (screenshot)
![schedule_card](https://user-images.githubusercontent.com/69635997/98361368-b35f9380-202b-11eb-9912-2f3e2620fe09.PNG)

Thank you for the very fast fix :)

**Comment by KTibow** [comment]
Consider closing? If you still think it's an issue, feel free to keep it open.

**Comment by TheNexusCore** [comment]
Forgot to close ^^

**Comment by nielsfaber** [comment]
Reopening, as a reminder to myself to fix the translation in a future update...

**Comment by nielsfaber** [comment]
Made some progress here (finally):
![Capture8](https://user-images.githubusercontent.com/16861605/98826390-e643c600-2435-11eb-8f2c-f634b7beb427.PNG)

Will be fixed in the next release 👍

**Comment by nielsfaber** [comment]
Fixed in v1.9.5. Closing.
